.github/scripts/openssl/install-dependencies.sh

```diff
@@ -2,7 +2,7 @@
 
 source `dirname ${BASH_SOURCE[0]}`/../config.sh
 
-echo "::group::Install OpenSSL Dependencies"
+echo "::group::Install OpenSSL dependencies"
```

Comment:
Should it be really changed? :)

Reply:
This makes the code style consistent as well.